src/composer.json

```diff
@@ -32,5 +35,11 @@
             "Fossology\\Decider\\UI\\": "decider/ui",
             "Fossology\\UI\\Page\\": "www/ui/page"
         }
+    },
+    
+    "config": {
+        "platform": {
+            "php": "5.3.3"
```

Comment:
Without this configuration, composer update would install package versions which require php>=5.4
We should answer question https://github.com/fossology/fossology/issues/585 soon.